relay-general/src/protocol/session.rs

```diff
@@ -117,6 +125,113 @@ impl SessionUpdate {
     }
 }
 
+#[allow(clippy::trivially_copy_pass_by_ref)]
+fn is_zero(val: &u32) -> bool {
+    *val == 0
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct AggregateItem {
```

Comment:
_nit_: Most types are named `Session*`, except this one. Can we consolidate this?